_docs/principles/accessibility.md

```diff
@@ -198,61 +299,67 @@ Helix minimum spacing standards require: 
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Provide easily identifiable feedback
+### Don't use tables for layout
 
-Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and provide suggestions to help fix issues whenever errors are returned. Prominently present any important feedback that requires user action.
+Do not use tables for formatting content or for anything other than presenting data. Because a screen reader reads every row and column of a table, including secondary text or data in additional nested rows can make interpreting the contents of a table difficult for the user.
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-#### Failed text inputs
+</section>
 
-There are many acceptable ways to make text inputs visually accessible. If you're indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red to draw the eye and doubles in thickness from 1px to 2px to provide another distinguishable visual cue that is not reliant on color.
+<section class="static-section"  markdown="1">
 
-It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error an state.
+<div class="hxRow"  markdown="1">
 
-{% endcolumn %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+## Interactivity guidelines
 
-{% figure [caption:"Default state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg"/>
-{% endfigure %}
+Consider the following page layout guidelines when you construct an accessible UI.
 
-{% endcolumn %}
+### Ensure that pointer targets are large enough
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+Pointer targets that are activated by a mouse, trackball, or stylus should be at least **Do we have a size?**. Large pointer targets can be helpful to users who have difficulty with fine motor skills.
```

Comment:
44x44px is general Best practice for tappable area, but we don't really have that defined per se...Check with @CITguy and @araiford

Reply:
Done. Settled on 44 x 44 px